refinery/core/utils.py

```diff
@@ -15,6 +15,7 @@
 from django.utils import timezone
 
 from celery.task import task
+from constants import REFINERY_SOLR_DOC_LIMIT
```

Comment:
`import constants` and move it into the "local" import section below since this is not a 3rd party module